src/ChildProcess.gren

```diff
@@ -37,7 +38,7 @@ This module allow you to spawn child processes.
 
 ## Spawning processes
 
-@docs SpawnOptions, Connection, defaultSpawnOptions, spawn, spawnWithDefaultOptions
+@docs SpawnOptions, Connection, defaultSpawnOptions, spawn, spawnWithDefaultOptions, spawnAndNotifyOnExit
```

Comment:
I went through a few rounds with this and kinda lost the plot with the name. I agree `spawnAndNotifyOnExit` is better since it describes exactly, and only, what it's doing. I'll change it.